src/bin/miri.rs

```diff
@@ -125,27 +124,27 @@ fn after_analysis<'a, 'tcx>(
 }
 
 fn init_early_loggers() {
-    // Notice that our `extern crate log` is NOT the same as rustc's!  So we have to initialize
-    // them both.  We always initialize miri early.
+    // Notice that our `extern crate log` is **not** the same as rustc's! So, we have to initialize
```

Comment:
I read this "So" just like "Thus", so there shouldn't be any comma.

Reply:
This is a standard grammatical rule. There needs to be a comma when a conjunction begins a sentence (which is something the real prescriptivists hate, but let's ignore that here).

Reply:
I'd actually suggest rephrasing the comment entirely - its word choice (imperative "notice", "early" rather than "first") is unidiomatic, and its pacing is choppy. I'd propose something like this:

> Note that our `extern crate log` is NOT the same as rustc's; as a result, we have to initialize them both, and we always initialize miri's first.

Reply:
@solson Consistency with the Rustc codebase, where `not` is always bold-emphasised rather than italic-emphasised.

Reply:
@eternaleye I concur.